### ISSUE TITLE
After migrating, HOOBS never starts enough to...Do anything. Can't load in the browser, can't log in via SSH.

### ISSUE THREAD

**Issue by ColinScattergood** [hoobs-org/HOOBS#1328]
**Description**
After resolving some dumb issues (using port 80 instead of port 8080 and stopping NGINX) I was able to migrate to HOOBS 4. However, after shutting down, it never seemed to come back up. I can't log in via the browser (it just churns forever) and I can't SSH (same result.) I'm at a loss for what next step to take w/ it.

**Version**
3.3.7 -> 4

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [X] Yes [3.3.6->3.3.7 prior to going to 4]
- [ ] No

**Pervious version**
3.3.7

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [ ] No
- [X] Maybe? If so it was a long time ago I honestly don't recall. I've been on HOOBS for a pretty long time.

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [X] Custom Install

**If this is a custom install, what's your operating system?**
Please let us know what operating system and version you are using. Ex. Fedora 30 or macOS Catalina.
Raspbian GNU/Linux v10

**List your plugins**
Please include a list of the plugins you are using. We need to be able to find it on NPM. If the plugin is no published, please include a link to the repository.
- Canary v1.0.3
- Camera FFMPEG v3.1.2
- myQ v2.6.2
- Weather Plus v3.2.6

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
{
"server": {
"polling_seconds": 5,
"origin": "*",
"home_setup_id": "X-HM://0023ISYWYA8CB",
"port": 80,
"autostart": 0
},
"client": {
"temp_units": "fahrenheit",
"locale": "en",
"theme": "hoobs-light",
"inactive_logoff": 30,
"default_route": "status",
"country_code": "US",
"postal_code": "08055",
"latitude": "HOMELAT",
"longitude": "HOMELONG"
},
"bridge": {
"name": "HOOBS",
"pin": "HOMEKITPIN",
"port": 51826,
"username": "USERNAME"
},
"description": "Homebridge server, managing garage doors and other light non-Homekit appliances.",
"ports": {},
"plugins": [
"homebridge-camera-ffmpeg",
"homebridge-canary",
"homebridge-weather-plus",
"homebridge-myq"
],
"accessories": [
{
"name": "Canary",
"serial": "C100K1615631",
"password": "PASSWORD
"model": "AllInOne",
"username": "USERNAME", "accessory": "Canary",
"pollingInterval": 300
}
],
"platforms": [
{
"name": "Camera-FFmpeg",
"plugin_map": {
"plugin_name": "homebridge-camera-ffmpeg"
},
"platform": "Camera-ffmpeg",
"cameras": [
{
"model": "WyzeCam v2",
"manufacturer": "Wyze",
"videoConfig": {
"maxBitrate": 299,
"maxFPS": 15,
"stillImageSource": "-i NETWORKADDRESS -vframes 1 -r 1", "motion": true,
"vcodec": "copy",
"additionalCommandline": "-protocol_whitelist https,crypto,srtp,rtp,udp",
"source": "-i NETWORKADDRESS",
"maxStreams": 2,
"maxWidth": 1280,
"maxHeight": 720,
"audio": true
},
"name": "Playroom"
}
]
},
{
"platform": "WeatherPlus",
"plugin_map": {
"plugin_name": "homebridge-weather-plus"
},
"units": "si",
"interval": 4,
"stations": [
{
"displayName": "TOWN NAME",
"service": "openweathermap",
"key": "KEY",
"forecast": [
0
],
"locationGeo": [
LAT,
LONG
]
}
]
},
{
"platform": "myQ",
"plugin_map": {
"plugin_name": "homebridge-myq"
},
"email": "EMAIL",
"password": "PASSWORD",
"name": "myQ"
}
],
"interfaces": [],
"package_manager": "npm",
"system": "hoobs"
```

**Post your log**
You can get the log from the interface. This is in the Log section.

```
No relevant log that I can access.
```

**Additional context**
N/A

**Comment by mkellsy** [comment]
This should help you
https://support.hoobs.org/docs/60e5b9026c19a6a5ca8ead7a

**Comment by ColinScattergood** [comment]
> This should help you
> https://support.hoobs.org/docs/60e5b9026c19a6a5ca8ead7a

Thanks. Walked through this, got all the way to the end and when and did the hbs-migrate t command, it ran and I selected Hoobs 3, it stated Nothing to Migrate. I then went looking for my hoobs data and...It appeared to be gone. That said, given the instructions, I'd just installed HOOBS 4. Which seems to run if I launch hoobsd, however, I can't load the interface in the browser. Any thoughts? At this point I'm kind of thinking...I had 4 plugins...It sucks but it would prob be faster to just wipe it and start from scratch.
